app/models/user.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import Column, String, Enum, Integer, DateTime, ForeignKey
+from sqlalchemy.orm import relationship
+from enum import Enum as PyEnum
+from database import Base
+
+
+class Role(str, PyEnum):
+    admin = "admin"
+    manager = "manager"
+    employee = "employee"
+
+class UserDB(Base):
+    __tablename__ = "users"
+
+    id = Column(Integer, primary_key=True, index=True)
+    email_address = Column(String, unique=True, index=True)
```

Comment:
this is fine but keep in mind index on small collections makes things slower, but here it can be fine